file/config/config.exs

```diff
@@ -0,0 +1,3 @@
+import Config
+
+config :opentelemetry, traces_exporter: :none
```

Comment:
Is this really needed? I would opt for removing opentelemetry from the FIle endpoint if there are any calls there

Reply:
Unfortunately, yes, it is needed. It is because `FileEndpoint` depends on `WebRTCEndpoint`, which depends on `membrane_webrtc_plugin` and some other plugins from which the Opentelemetry wasn't removed yet.

Reply:
Fixed.